README.md

```diff
@@ -1,3 +1,101 @@
+
+  
+
+Hello and welcome to **Starc's** GitHub page. Story Architect is a project created by the authors of an open source screenwriting tool [**Kit Scenarist**](https://github.com/dimkanovikov/KITScenarist) and is currently under development with its current release being alpha. It's an advanced text editor for script writers working professionally in film, theatre, radio or writing for comic books.
+
+  
+
+## Installation
+
+  
+
+**Starc** is available for Windows, Mac and Linux. You can download it from [**starc.app**](https://starc.app/download) website.
+
+  
+
+## Supported formats
+
+  
+
+Starc has its own *.starc* file format, but also allows you to import files using Final Draft's *.fdx* format as well as *.docx*, Fountain and importing projects from **Kit Scenarist**.
+
+  
+
+>  **Note:** Currently, importing projects from KIT Scenarist only allows importing the script. Importing other data, such as text documents and images will be supported in future releases.
+
+## Language support
+
+**Starc** currently supports following languages:
+
+- Azerbaijani
+- Croatian
+- Galician
+- German (de)
+- Portuguese (Brazil)
+- Romanian (Romania)
+- Russian (ru) 
+- Spanish (es) 
+- Ukrainian (uk) 
+- French (fr) 
+- Hebrew (he) 
+- Slovenian (sl) 
+- Belarusian (be) 
+- Hungarian (hu) 
+- Turkish (tr) 
+- Catalan (ca) 
+- Hindi (hi) 
+- Polish (pl) 
+- Italian (it) 
+- Danish (Denmark) 
+- Tagalog (Philippines) 
+- Portuguese (Portugal) 
+- Persian (fa) 
+- Indonesian (id) 
+- Esperanto (eo) 
+- Arabic (ar) 
+- Dutch (nl) 
+- Filipino (Philippines) 
+- Tamil (ta)
+- Telugu (te) 
+
+If you wish to add another language, please consult our [Wiki page on how to do it yourself](https://github.com/story-apps/starc/wiki/How-to-add-the-translation-of-Story-Architect-to-your-native-language-or-improve-the-existing-version?).
+
+
+## Features
+
+**Starc** supports writing for film, television, comic books, stage and radio. With the addition of supporting multiple formats, there are also templates for you to use, such as **Final Draft Screenplay Template**. 
+
+### Modules
+
+Alongside these, **Starc** uses modules for research and for writing. You can add text documents, mind maps and images in addition to your script. If you're writing in episodic format, you can add multiple scripts to one project and use existing assets.
+
+Starc can also help you track your characters, locations, etc. with its statistics module, as well as edit them and add additional information about them (for example, characters real name) which can be useful in pre-production.
+
+![Modules](./img/modules.png)
+
+You can also add things like title page for your script, movie poster and other information that might be useful to other crew members.
+
+## Cloud sync
+
+**NOTE:** This feature is in development and will be added soon.
+
+## Build instructions
+
+### Linux
+
+    git clone https://github.com/story-apps/starc
+    cd starc/build
+    ./make-starc-app-image.sh
```

Comment:
It's not a build script :) It's just a helper script which we use for build AppImage file inside the GitHub Action.

Build instruction here should be something like

```
git clone --recurse-submodules https://github.com/story-apps/starc
cd starc
qmake && make
```

or just refer to the GitHub actions [build script](https://github.com/story-apps/starc/blob/master/.github/workflows/build_starc_app.yml).